src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -364,18 +369,28 @@ object LorenzUtils {
         return this
     }
 
-    private var lastMessageSent = 0L
+    private var lastMessageSent = SimpleTimeMark.farPast()
+    private val sendQueue: Queue<String> = LinkedList()
+
+    @SubscribeEvent
+    fun sendQueuedChatMessages(ignored: LorenzTickEvent) {
+        val thePlayer = Minecraft.getMinecraft().thePlayer
+        if (thePlayer == null) {
+            sendQueue.clear()
+            return
+        }
+        if (lastMessageSent.passedSince() > 300.milliseconds) {
+            lastMessageSent = SimpleTimeMark.now()
+            thePlayer.sendChatMessage(sendQueue.poll() ?: return)
```

Comment:
i think this should also check if there is something in the queue so that when a command is sent it is not waiting for up to 300ms